api/lib/domain/usecases/create-organizations-with-tags-and-target-profiles.js

```diff
@@ -103,7 +104,11 @@ async function _createOrganizations({ transformedOrganizationsData, domainTransa
       throw new InvalidInputDataError(`User with ID "${createdByUserId}" does not exist`);
     }
 
-    throw error;
+    if (error instanceof DomainError) {
+      throw error;
+    }
+
+    throw new DomainError(error.message);
```

Comment:
Suggestion de modification du message du commit `feat(api): map error to domain error` :

```
feat(api): map unhandled errors to domain errors


to have the error message directly displayed on front apps
```